sympy/combinatorics/homomorphisms.py

```diff
@@ -417,3 +420,125 @@ def block_homomorphism(group, blocks):
     images = {g: Permutation([b[p[i]^g] for i in identity]) for g in group.generators}
     H = GroupHomomorphism(group, codomain, images)
     return H
+
+def group_isomorphism(G, H, isomorphism=False):
+    '''
+    Compute an isomorphism between 2 given groups.
```

Comment:
If this is how we describe it, then `isomorphism` should be `True` by default. Possibly worth actually having a little function like
```
def is_isomorphic(G, H):
    return group_isomorphism(G, H, isomorphism=False)
```
just because that would be shorter to use.